test/common.js

```diff
@@ -598,8 +598,7 @@ exports.WPT = {
     try {
       fn();
     } catch (err) {
-      if (err instanceof Error)
-        err.message = `In ${desc}:\n  ${err.message}`;
+      console.error(`In ${desc}:`);
```

Comment:
@joyeecheung, calling `Error.captureStackTrace()` there will only capture the stack of `WPT.test`, not the actual stack from `err`.